sdk/core/azure-core/src/http/request.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <string>
+
+#include <http/http.hpp>
+
+using namespace azure::core::http;
+using namespace std;
```

Comment:
I believe it's a bad practice to `using namespace std`.

Reply:
Will fix it.
thanks.